### DISCUSSION TITLE
Pytest-Jenkins-Docker-Allure Integration

### DISCUSSION THREAD

**Discussion by ShankarSampath** [allure-framework/allure2#1856] [General Discussion]
Hi All,

I'm not getting alllure results using jekins pipeline for pytest program wit Docker.

* Had build image in docker using Ec2 Instance.
* Execute the script using jenkins pipeline.
* Still im getting error. PFB attached screenshots for reference and help me out. Thanks in advance.

*** JenkinsFile & Docker File**
![image](https://user-images.githubusercontent.com/77034724/207024601-e291b9b7-ac8a-4bed-84aa-635014b20f95.png)
*** Workspace**
<img width="669" alt="image" src="https://user-images.githubusercontent.com/77034724/207025001-f8e278ab-d4dc-450e-b8a0-a8cf9ed450d5.png">
*** Console Log**
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/77034724/207025098-b7534dd9-5d7f-40c8-9d65-d1140e7b03ab.png">
* **Allure** 

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/77034724/207025162-b62668c0-cf43-43b5-aa04-9e704b8b6140.png">

**Comment by cheshi-mantu** [comment]
@ShankarSampath 
Hello sir,
please refer to the allure framework documentation: https://docs.qameta.io/allure-report/#_pytest
I do not see you've installed allure-pytest and I'm not sure you're using the adaptor in your code (magic does not happen without the adaptor, see 6.1.2. Usage).
And I do not see any instructions allowing the files from container to land into allure-results folder which resides outside of the container.

**Reply by ShankarSampath** [reply]
Thanks now it  works
